src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/OpenSslX509ChainProcessor.cs

```diff
@@ -1069,6 +1085,10 @@ private static ArraySegment<char> Base64UrlEncode(ReadOnlySpan<byte> input)
 
             elementStatus.Add(chainStatus);
             AddUniqueStatus(overallStatus, ref chainStatus);
+            if (statusFlag == X509ChainStatusFlags.NotSignatureValid)
```

Comment:
```suggestion

            if (statusFlag == X509ChainStatusFlags.NotSignatureValid)
```

I like blank lines before control flows :smile: